bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/dao/PatientDao.java

```diff
@@ -19,6 +20,12 @@
                                                        String programAttributeFieldName, String[] addressSearchResultFields,
                                                        String[] patientSearchResultFields, String loginLocationUuid, Boolean filterPatientsByLocation, Boolean filterOnAllIdentifiers);
 
+    List<PatientResponse> getSimilarPatientsUsingLuceneSearch(String identifer, String name, String gender, String customAttribute,
```

Comment:
Do we really have to add another method here? The DAO isn't an external-facing interface so we should just be able to add a parameter to the above, right?